### DISCUSSION TITLE
Shelly 1PM model (those that are not PLUS) is it compatible?

### DISCUSSION THREAD

**Discussion by litio2001** [camueller/SmartApplianceEnabler#402] [Q&A]
Hi!
It is compatible SAE with "Shelly 1PM" devices (those that are not PLUS)

**Comment by camueller** [comment]
It should work with any Shelly as long as the URL is correct. Simply try any URL in your browser and you will know if  it is working.
Please see https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SEMP_EN.md

**Reply by litio2001** [reply]
thanks you for quick response!
